src/Microsoft.Health.Fhir.Core/Features/Search/Converters/CodeToTokenSearchValueTypeConverter.cs

```diff
@@ -14,11 +14,11 @@ namespace Microsoft.Health.Fhir.Core.Features.Search.Converters
     /// <summary>
     /// A converter used to convert from <see cref="Code"/> to a list of <see cref="TokenSearchValue"/>.
     /// </summary>
-    public class CodeNodeToTokenSearchValueTypeConverter : FhirNodeToSearchValueTypeConverter<TokenSearchValue>
+    public class CodeToTokenSearchValueTypeConverter : FhirTypedElementToSearchValueTypeConverter<TokenSearchValue>
```

Comment:
Do we even need type in there anymore on the base?
```suggestion
    public class CodeToTokenSearchValueTypeConverter : FhirTypedElementToSearchValueConverter<TokenSearchValue>
```